docs/developer-docs/latest/developer-resources/database-apis-reference/rest-api.md

```diff
@@ -0,0 +1,645 @@
+
+:::: tabs card
+
+::: tab Collection Type
+
+<div id="endpoint-table">
+
+<!-- ? is `pluralApiId` the value declared with `info.pluralName` in the model? -->
+
+| Method   | URL                                             | Description                                                           |
+| -------- | ----------------------------------------------- | --------------------------------------------------------------------- |
+| `GET`    | `/api/:pluralApiId`                             | [Get a list of entries](#get-entries)                             |
+| `POST`   | `/api/:pluralApiId`                             | [Create an entry](#create-an-entry)                                |
+| `GET`    | `/api/:pluralApiId/:documentId`                 | [Get an entry](#get-an-entry)                                     |
+| `PUT`    | `/api/:pluralApiId/:documentId`                 | [Update an entry](#update-an-entry)                                |
+| `DELETE` | `/api/:pluralApiId/:documentId`                 | [Delete an entry](#delete-an-entry)                                |
+| `POST`   | `/api/:pluralApiId/actions/:action`             | Actions on the collection of documents (bulk actions, custom action…) |
+| `POST`   | `/api/:pluralApiId/:documentId/actions/:action` | Actions on a specific document                                        |
+
+</div>
+
+:::
+
+::: tab Single Type
+
+<div id="endpoint-table">
+
+| Method   | URL                                 | Description                                 |
+| -------- | ----------------------------------- | ------------------------------------------- |
+| `GET`    | `/api/:singularApiId`                 | [Get an entry](#get-an-entry)             |
+| `PUT`    | `/api/:singularApiId`                 | [Update an entry](#update-an-entry)    |
+| `DELETE` | `/api/:singularApiId`                 | [Delete an entry](#delete-an-entry)        |
+| `POST`   | `/api/:singularApiId/actions/:action` | Actions on the single type (custom action…) |
+
+</div>
+
+:::
+
+::::
+
+::::: details Examples
+
+:::: tabs card
+
+::: tab Collection Type
+
+`Restaurant` **Content Type**
+
+<div id="endpoint-table">
+
+| Method | URL                      | Description               |
+| ------ | ------------------------ | ------------------------- |
+| GET    | `/api/restaurants`       | Get a list of restaurants |
+| GET    | `/api/restaurants/:id`   | Get a specific restaurant |
+| POST   | `/api/restaurants`       | Create a restaurant       |
+| DELETE | `/api/restaurants/:id`   | Delete a restaurant       |
+| PUT    | `/api/restaurants/:id`   | Update a restaurant       |
+
+</div>
+
+:::
+
+::: tab Single Type
+
+`Homepage` **Content Type**
+
+<div id="endpoint-table">
+
+| Method | URL             | Description                 |
+| ------ | --------------- | --------------------------- |
+| GET    | `/api/homepage` | Get the homepage content    |
+| PUT    | `/api/homepage` | Update the homepage content |
```

Comment:
update/create actually. Since we can delete it, it makes sense to me to specify how we also "create" it.